src/main/resources/stories/runtime-bundle/task-actions.story

```diff
@@ -11,4 +11,5 @@ When the user starts a random process
 And the user claims a task
 And the user completes the task
 Then the status of the process is changed to completed
+And the status of the process is changed to completed by querying
```

Comment:
@constantin-ciobotaru i reported this line also on Diego's PR can you please have a look at this comment and see if it makes sense to you? Thanks.
https://github.com/Activiti/activiti-cloud-acceptance-tests/pull/1#issuecomment-365982825

Reply:
@ffazzini Agree, I'll update the PR.